rpc/verif_proxy.go

```diff
@@ -6,14 +6,15 @@ package rpc
 import (
 	"context"
 	"fmt"
-)
 
```

Comment:
Thanks for this. This might have been some linting issue right?

Reply:
yes 👍